app/assets/stylesheets/admin/global/_forms.sass

```diff
@@ -24,7 +24,7 @@ fieldset
 select[multiple=multiple],
 textarea
   margin-bottom: ($base-spacing / 2)
-  padding: ($base-spacing / 3)
+  padding: 6px 20px
```

Comment:
IMO 20px left and right padding is a lot. Looks like something is missing / has failed to load..

![screen shot 2016-01-11 at 17 07 42](https://cloud.githubusercontent.com/assets/2037851/12229780/d955b35c-b885-11e5-805b-7e84b6fa9643.png)